src/components/Welcome.jsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Welcome = () => <p>Hello, user!</p>;
+const Welcome = (props) =>
+  <p>{`За ${props.gameTime} минут нужно ответить на все вопросы. Можно допустить ${props.maxErrorCount} ошибки`}</p>;
```

Comment:
 А вот так можно сделать?
```suggestion
  <p>За {props.gameTime} минут нужно ответить на все вопросы. Можно допустить {props.maxErrorCount} ошибки</p>;
```